specification/vmware/resource-manager/readme.md

```diff
@@ -21,7 +21,23 @@ These are the global settings for the VMware Solution API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2020-03-20
+tag: package-2020-07-17-preview
+```
+
+### Tag: package-2020-07-17-preview
+
+These settings apply only when `--tag=package-2020-07-17-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2020-07-17-preview'
+input-file:
+- Microsoft.AVS/preview/2020-07-17-preview/vmware.json
+directive:
+  - suppress: R3020
+    from: vmware.json
+    reason: Microsoft.AVS was chosen over Microsoft.AzureVMwareSolution
+  - suppress: R3010
```

Comment:
The R3010 is warning , no needs to suppress , BTW you should follow the suppression process [here](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/85/Swagger-Suppression-Process) to add a suppression:

Reply:
thanks